lib/fluent/plugin/filter_record_transformer.rb

```diff
@@ -194,6 +194,7 @@ def preprocess_map(value, force_stringify = false)
 
       def prepare_placeholders(placeholder_values)
         placeholders = {}
+        reserved_keys = Set.new(placeholder_values.keys)
```

Comment:
Why is this `Set` needed?
Does `placeholder_values.include?(k)` have a problem?